<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerForm</class>
 <widget class="QWidget" name="ProductManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Product Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Stock</string>
         </property>
        </column>
       </widget>
       <widget class="QToolBox" name="toolBox">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="inputPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>241</width>
           <height>379</height>
          </rect>
         </property>
         <attribute name="label">
          <string>&amp;Input</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="PidLabel">
                <property name="text">
                 <string>I&amp;D</string>
                </property>
                <property name="buddy">
                 <cstring>PidLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="PidLineEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="ProductNameLabel">
                <property name="text">
                 <string>&amp;ProductName</string>
                </property>
                <property name="buddy">
                 <cstring>nameLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="nameLineEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="priceLabel">
                <property name="text">
                 <string>&amp;Price</string>
                </property>
                <property name="buddy">
                 <cstring>PricelineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="PricelineEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="stockLabel">
                <property name="text">
                 <string>&amp;Stock</string>
                </property>
                <property name="buddy">
                 <cstring>quantitylineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="quantitylineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>138</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="addPushButton">
                <property name="text">
                 <string>&amp;Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="modifyPushButton">
                <property name="text">
                 <string>&amp;Modify</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="searchPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>241</width>
           <height>379</height>
          </rect>
         </property>
         <attribute name="label">
          <string>&amp;Search</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeWidget" name="searchTreeWidget">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Product Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Price</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Stock</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QComboBox" name="searchComboBox">
                <item>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Product Name</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Price</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Stock</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="searchLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="searchPushButton">
              <property name="text">
               <string>S&amp;each</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="producttableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
